sql/moz-fx-data-marketing-prod/ga_derived/www_site_metrics_summary_v2/query.sql

```diff
@@ -0,0 +1,71 @@
+-- Query for ga_derived.www_site_metrics_summary_v2
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+WITH site_data AS (
+  SELECT
+    PARSE_DATE('%Y%m%d', event_date) AS date,
+    device.category AS device_category,
+    device.operating_system AS operating_system,
+    device.web_info.browser AS browser,
+    device.language AS language,
+    geo.country AS country,
+    traffic_source.source AS source,
+    traffic_source.medium AS medium,
+    collected_traffic_source.manual_campaign_name AS campaign,
+    collected_traffic_source.manual_content AS ad_content,
```

Comment:
Are these attribution fields the same ones we pulled from UA (universal analytics)? I want to make sure we've manually verified they match

Reply:
You're a mind reader - I was literally struggling with this earlier today and tried to figure it out but couldn't figure out which ones were the right ones to use because there are 2 different columns in GA4 for source and medium, and they both are similar but different - setting up time to pick your brain on how to best decide which ones to use. Thanks!